Resources/Prototypes/Catalog/VendingMachines/Inventories/magivend.yml

```diff
@@ -9,6 +9,9 @@
     ClothingHeadHatVioletwizard: 3
     ClothingOuterWizardViolet: 3
     ClothingShoesWizard: 9
+  contrabandInventory:
+    ToyFigurineWizard: 1
+    ToyFigurineWizardFake: 1
```

Comment:
I wouldn't include the wizard ones, iirc the wizard ones are for a spell we don't have yet where people get turned into figurines.

Reply:
Figurines removed.